packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.controller.js

```diff
@@ -0,0 +1,148 @@
+import get from 'lodash/get';
+
+export default class XdslMeetingCtrl {
+  /* @ngInject */
+  constructor($scope, $translate, OvhApiXdsl, TucToast) {
+    this.$scope = $scope;
+    this.$translate = $translate;
+    this.OvhApiXdsl = OvhApiXdsl;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    this.select = {
+      meetingSlots: {
+        fakeMeeting: false,
+        slot: null,
+      },
+    };
+
+    this.showMeetingSlots = false;
+    this.meetingSlots = {};
+
+    this.loading = true;
+    this.meetings = [];
+
+    this.errorMessage = '';
+    this.successMessage = '';
+
+    // Retrieve list of meetings available
+    return this.OvhApiXdsl.v6()
```

Comment:
Maybe this part could be _resolved_ as it's done for the `serviceName` ?

Reply:
Sorry but I would prefer not.

Reply:
What motivates this choice? Is there a problem when using `resolve`s ? (We could help you if you encounter some issues.)
It is one of our best practices to use resolves rather than making API calls directly in the controller.